src/customTheme.ts

```diff
@@ -101,36 +103,81 @@ const tertiaryTextDarkmode = coolGray['400']
 // ***** Collect constants, put them in a customTheme *****
 
 const themeConfig: ThemeConfig = {
-  initialColorMode: 'light',
   useSystemColorMode: !!process.env.NEXT_PUBLIC_FEATURE_FLAG_COLOR_MODE,
 }
 
-const customTheme = extendTheme({
-  colors: {
-    blue: blueColor,
-    primary: primaryColor,
-    secondary: blueColor,
-    coolGray,
-    fuchsia,
-    sky,
-    primaryTextLightmode,
-    secondaryTextLightmode,
-    tertiaryTextLightmode,
-    primaryTextDarkmode,
-    secondaryTextDarkmode,
-    tertiaryTextDarkmode,
-  },
-  fonts: {
-    heading: '"Inter", sans-serif;',
-    body: '"Inter", sans-serif;',
-    mono: '"Roboto Mono", monospace;',
-  },
-  styles: {
-    global: {
-      'html, body, #__next': { height: '100%' },
+const customTheme = extendTheme(
+  {
+    colors: {
+      blue: blueColor,
+      primary: primaryColor,
+      secondary: blueColor,
+      coolGray,
+      fuchsia,
+      sky,
+      primaryTextLightmode,
+      secondaryTextLightmode,
+      tertiaryTextLightmode,
+      primaryTextDarkmode,
+      secondaryTextDarkmode,
+      tertiaryTextDarkmode,
+    },
+    fonts: {
+      heading: '"Inter", sans-serif;',
+      body: '"Inter", sans-serif;',
+      mono: '"Roboto Mono", monospace;',
+    },
+    styles: {
+      global: {
+        'html, body, #__next': { height: '100%' },
+      },
+    },
+    config: themeConfig,
+    components: {
+      Link: {
+        baseStyle: (props: Dict) => {
+          return { color: mode('fuchsia.700', 'fuchsia.400')(props) }
+        },
+      },
+      Button: {
+        variants: {
+          cta: (props: Dict) => {
+            return {
+              fontWeight: 'black',
+              fontSize: '2xl',
+              letterSpacing: 'tight',
+              p: 6,
+              size: 'lg',
+              boxShadow: '0px 4px 4px rgba(0, 0, 0, 0.25)',
+              borderRadius: '2xl',
+              bg: mode('fuchsia.900', 'fuchsia.200')(props),
+              color: mode('white', 'fuchsia.900')(props),
+              _hover: {
+                bg: mode('fuchsia.700', 'fuchsia.100')(props),
+                cursor: 'pointer',
+              },
+              _active: {
+                bg: mode('fuchsia.800', 'fuchsia.200')(props),
+                boxShadow: '0px 2px 2px rgba(0, 0, 0, 0.25) !important',
+              },
+            }
+          },
+        },
+      },
+      Container: {
+        variants: {
+          fluid: {
+            maxWidth: 'container.xl',
+          },
+        },
+      },
     },
   },
-  config: themeConfig,
-})
+  withDefaultColorScheme({ colorScheme: 'primary' }),
+  withDefaultColorScheme({
+    colorScheme: 'gray',
+    components: ['Code', 'BlockQuote'],
+  })
```

Comment:
I'm confused about the `withDefaultColorScheme` property we have here.

Is there a reason you're using `withDefaultColorScheme({ colorScheme: 'primary' }),`,  instead of   withDefaultColorScheme({ colorScheme: 'fuchsia' }),

`primary` is the old color scheme. So I worry that we're now mixing old design with new Figma design colors, in such a way that it will be hard to see what colors come from what part of the code???

Or maybe this will get fixed up later?

Reply:
I set "primary" as the default color scheme just to keep the current one. If we want to do the change now, rather than set the default color schema to "fuchsia", I'll set the "primary" token to "fuchsia" and remove the deprecated colors.

My idea was to do this in another PR and just extract the custom components into the extended Chakra theme here, but as you prefer.

Reply:
> My idea was to do this in another PR and just extract the custom components into the extended Chakra theme here, but as you prefer.

Oh that sounds easier to review and merge, so let's do it in another PR. 😉

Reply:
I just remembered we have #243 to do this, so I'll take that ticket after this one!

Reply:
Sounds good to me. I'll resolve the current conversations and give my approval so we can merge this. 😄